### DISCUSSION TITLE
Releases within a monorepo

### DISCUSSION THREAD

**Discussion by rawkode** [community/community#86745] [Repositories]
### Select Topic Area

Product Feedback

### Body

The releases page becomes rather unwieldy when a repository is a monorepository and contains many projects that have any sort of release artifact.

It would be really nice if the releases page could be filtered by projects, based on tag prefixes.

For example, if I have a server and a client within the same repository:

- server/v1.1.1
- client/v0.1.2

The releases could be filtered by the two prefixes, client and server.

**Comment by yordis** [comment]
Maybe introducing the concept of "Release Channels." The end-user decides whatever those channels are. It is just a text field adding a second dimension to be able to group by.

In your case, it could be "server" or "client," probably following the package name.
